src/main/java/lulu/task/Event.java

```diff
@@ -0,0 +1,66 @@
+package lulu.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+public class Event extends Task {
+    private LocalDate fromDate;
+    private String from;
+    private LocalDate toDate;
+    private String to;
+
+    public Event(String description, String from, String to) {
+        super(description);
+        try {
+            if (from.split("-").length == 3) {
```

Comment:
Perhaps add a comment explaining why the specific numbers are used or use named constants?